Araali Tools/goshelly-server-helm/templates/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: goshelly
+  labels:
+    name: goshelly
+
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: goshelly
+provisioner: docker.io/hostpath
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: goshelly
+  namespace: goshelly
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteMany
+  hostPath:
+    path: "/mnt/goshelly"
+  storageClassName: goshelly
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: goshelly
+  namespace: goshelly
+spec:
+  storageClassName: goshelly
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+  volumeMode: Filesystem
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: goshelly
+  namespace: goshelly
+spec:
+  volumes:
+    - name: goshelly
+      persistentVolumeClaim:
+        claimName: goshelly
+  containers:
+    - name: goshelly
+      image: ubuntu:latest
+      command: ["/bin/sleep", "3650d"]
```

Comment:
[ "while true; do sleep 30; done;" ], or 

["tail", "-f", "/dev/null"]

The benefit of the first one is that its in a loop, if the tail finishes/crashes for some reason (it shouldnt), it will exit. You can combine the two approaches:

```
[ "while", "true;", "tail", "-f",  "/dev/null;", "done;" ]
```